### ISSUE TITLE
Copy/Paste keyboard shortcuts not working in MacOS Chrome

### ISSUE THREAD

**Issue by SandyChapman** [jgraph/drawio#949]
### Preflight Checklist

* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Describe the bug**

Per discussion on #843 opening this ticket. Neither Cmd+C, nor Cmd+V are working for me on MacOS Chrome.

**To Reproduce**

1. open a new diagram, 
2. add a box
3. try to copy and paste using cmd+c, cmd+v. 

No errors are printed to console. 
No extensions are enabled. 
Does not work in Incognito mode.

**Expected behavior**

Any element selected should be copied on "cmd+c" and a duplicate created on "cmd+v". Note: cmd+d for duplicate works as expected.

**Screenshots**

I've recorded a short video. You can see when I press cmd+c, cmd+v on the keyboard, the Mac OS menu bar's "edit" menu flickers showing the OS has handle the cmd+c and cmd+v hotkeys.

https://drive.google.com/file/d/1ocEhC2R7P70usdBmHKLcxYkwRkOJBKhA/view?usp=sharing

**draw.io version (In the Help->About menu of the draw.io editor):**

- draw.io version 13.1.12

**Desktop (please complete the following information):**

Google Chrome | 83.0.4103.61 (Official Build) (64-bit)
Revision | 94f915a8d7c408b09cc7352161ad592299f384d2-refs/branch-heads/4103@{#561}
OS | macOS Version 10.15.3 (Build 19D76)
JavaScript | V8 8.3.110.9 
User Agent | Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.61 Safari/537.36
Command Line | /Applications/Google Chrome.app/Contents/MacOS/Google Chrome --flag-switches-begin --flag-switches-end --enable-audio-service-sandbox

**Additional context**
See #843

**Comment by SandyChapman** [comment]
I've discovered an additional point for this ticket:

The problem only exists on my external USB keyboard. This appears to be happening only when you remap your modifier keys in the MacOS keyboard settings.

Additional steps to reproduce:

Open "System Preferences".
Select "Keyboard"
On the "Keyboard" panel, select "Modifier Keys..." in the bottom right.
Switch "Option Key" to be "Command" and "Command Key" to be "Option".
The expectation is now that your option key (on my USB keyboard this is "Alt") to work as cmd when you do cmd+c, cmd+v (or in this case alt+c, alt+v). However, neither the original key maps (option+c, option+v) nor the remapped keys work with diagrams.net.

Note: this is completely reproducible using the built-in keyboard following the same steps. An external USB keyboard is not required.

**Comment by SandyChapman** [comment]
Even stranger, if with the remapping, using my right "Alt" key works. It's just my left alt key that's failing and on the macbook keyboard, the right cmd key fails in the key combo. Using this site:

https://w3c.github.io/uievents/tools/key-event-viewer.html

It's clear that my USB keyboard's left and right alt keys are transposed. Pushing the right alt key is shown as the left key on the event viewer and vice versa.

In summary, draw.io is failing to detect "Right cmd+c, Right cmd+v" correctly, likely because these right and left cmd have different key codes, 91 vs 93. I think all that's needed to fix this bug is to add the additional event handlers for the right-side modifier keys.

**Comment by SandyChapman** [comment]
Poking the following people since they were looking at the linked issue last week:

@davidjgraph @alderg

**Comment by nitramwin** [comment]
Thanks for posting this @SandyChapman - I have been seeing this for a very long time as well (2+ years). Of note is that this occurs in Safari and Chrome so I imagine that the problem lies with interpreting the MacOS key mapping.

**Comment by davidjgraph** [comment]
Is this the right-side cmd key not working, but left-side does?

**Comment by andrebrantom** [comment]
The issue for me is only with a right-hand CMD-C/V on both an external USB keyboard (Kinesis Advantage 2) but also on my MacBook Pro's built-in keyboard. No remapping on either keyboard (Kinesis is remapped in its own hardware).

**Comment by alderg** [comment]
Thanks for the report. This will be fixed in the next release.

**Comment by janvonborstel** [comment]
@alderg I believe the "copy/paste style" shortcuts for latest chrome are broken, again :-(
at least for MACOS12.6.

**Comment by joseph-zhong** [comment]
+1, I've just upgraded to Sonoma and this issue seems to have come back, keyboard shortcuts for copy/cut/paste all don't work :(
